api/http/security/bouncer.go

```diff
@@ -244,6 +304,26 @@ func ExtractBearerToken(r *http.Request) (string, error) {
 	return token, nil
 }
 
+// extractAPIKey extracts the api key from the api key request header or query params.
+func extractAPIKey(r *http.Request) (apikey string, ok bool) {
```

Comment:
Using named return params here?

Reply:
I dont like this either.
Wasn't done by me though, was done by peer-reviewer - so let it pass.